src/providerhub/azext_providerhub/generated/_help.py

```diff
@@ -274,42 +347,47 @@
             service-id: Required. The service ID.
             component-id: Required. The component ID.
 
-      - name: --resource-access-policy
-        type: string
-        short-summary: 'The resource access policy.'
-        long-summary: |
-            Usage: --resource-access-policy "AcisReadAllowed, AcisActionAllowed"
 
-      - name: --opt-in-headers
-        type: string
-        short-summary: 'The opt-in headers.'
+            Multiple actions can be specified by using more than one --service-tree-infos argument.
+      - name: --subscription-state-override-actions --override-actions
```

Comment:
@kairu-ms some of the help entries are failing the unrecognized_help_parameter_rule but I'm not sure why these are considered invalid since I've included all the parameter names. Can you please help here?

Help-Entry: providerhub provider-registration create - The following parameter help names are invalid: --subscription-state-override-actions --override-actions | --providerhub-metadata-authorizations --metadata-authz | --providerhub-metadata-authentication --metadata-authn | --lighthouse-authorizations --lighthouse-auth
Help-Entry: providerhub resource-type-registration create - The following parameter help names are invalid: --resource-creation-begin --creation-begin | --resource-patch-begin --patch-begin | --authorization-action-mappings --auth-mappings | --required-features-policy --req-features-policy | --disallowed-action-verbs --dav | --subscription-state-rules --sub-state-rules | --resource-deletion-policy --deletion-policy | --subscription-state-override-actions --override-actions